### PR TITLE
Remove the last usage of obsolete package

### PR DESCRIPTION
**Pull request by baijum**
The github.com/pkg/errors package is no more maintaned:
https://github.com/pkg/errors/issues/245

Signed-off-by: Baiju Muthukadan <baiju.m.mail@gmail.com>

# Changes

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] [Docs](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#docs) 
  included if any changes are user facing
- [ ] [Tests](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#tests)
  included if any functionality added or changed. For bugfixes please include tests that can catch regressions
- [ ] Follows the [commit message standard](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#commits)

### CODE REVIEW

File: vendor/modules.txt [hunk 1]
```diff
@@ -128,7 +128,6 @@ github.com/operator-framework/api/pkg/lib/version
 github.com/operator-framework/api/pkg/operators
 github.com/operator-framework/api/pkg/operators/v1alpha1
 # github.com/pkg/errors v0.9.1
-## explicit
 github.com/pkg/errors
```
Comment (pmacik) [inline vendor/modules.txt:131]: Shouldn't these be also removed?

Comment (baijum) [reply vendor/modules.txt:131]: @pmacik That package is transitive dependency. For example `github.com/evanphx/json-patch` and `go.uber.org/zap` use it. Again these packages are also transitive dependencies. So, only the explicit dependency has been removed.

**Comment by pmacik** [comment]
@baijum please rebase on top of the current master to rebuild-the images for acceptance tests to match the acceptance tests scenarios. Otherwise, the acceptance tests would keep failing.

**Pull request merged**